app/models/user.rb

```diff
@@ -1,16 +1,28 @@
 class User < ApplicationRecord
-  validates :name, presence: true
-  validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
-
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  #  :confirmable
```

Comment:
- [ ] Please enable `:confirmable` so that the application asks for confirmation of email before signing up

Reply:
alright. I've just forget to remove the comment
Thanks